lib/view_model/events.dart

```diff
@@ -58,22 +58,23 @@ class EventsLogic extends ChangeNotifier {
     for (int i = 0; i < eventsDocIds.length; i++) {
       DocumentSnapshot<Object?> snap =
           await _collectionRef.doc(eventsDocIds[i]).get();
-      addInSpecificEvents(snap);
+      _specificEvents.add(addInSpecificEvents(snap));
     }
     stateOfSpecificEvents = StateOfSpecificEvents.loaded;
     notifyListeners();
   }
 
-  addInSpecificEvents(DocumentSnapshot<Object?> snap) {
-    _specificEvents.add(Events(
+ Events addInSpecificEvents(DocumentSnapshot<Object?> snap) {
```

Comment:
What is "specific" about the event?

Reply:
The name of the class is "Events" but it holds data for one "Event". 😞

Reply:
solved